docs/sources/setup/install/helm/install-microservices/_index.md

```diff
@@ -167,6 +167,10 @@ It is not recommended to run scalable mode with `filesystem` storage. For the pu
 
 After testing Loki with MinIO, it is recommended to configure Loki with an object storage provider. The following examples shows how to configure Loki with different object storage providers:
 
+{{< admonition type="note" >}}
+When deploying Loki using S3 Storage **DO NOT** use the default bucket names; `chunk`, `ruler` and `admin`. Choose a unique name for each bucket. For more information see the following [security update](https://grafana.com/blog/2024/06/27/grafana-security-update-grafana-loki-and-unintended-data-write-attempts-to-amazon-s3-buckets/).
```

Comment:
It's fine when using MinIO and we still use & ~recommend it there as it's easiest. Not sure if or how  to reflect that properly